CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.32.0-rc.0 / 2021-12-01
+
+This list of changes is relative to [v2.32.0-beta.0](https://github.com/prometheus/prometheus/releases/tag/v2.32.0-beta.0), which was a special 2.32 beta release to pre-publish the Prometheus Agent features.
+
+* [FEATURE] TSDB: Add support for forwarding exemplars in Agent mode.
+* [FEATURE] UI: Adapt web UI for Prometheus Agent mode. #9851.
```

Comment:
```suggestion
* [FEATURE] UI: Adapt web UI for Prometheus Agent mode. #9851
```

Reply:
Fixed this and the other comment, thanks!